### ISSUE TITLE
Feature Request: Nintendo Switch Pro Controller Support

### ISSUE THREAD

**Issue by harley-codes** [seblindfors/ConsolePort#95]
As the title suggests, it would be great to have proper support for this. Using other profile such as Xbox is confusing with the layout shift (Nintendo trying to be special). 

For someone trying to play via Steam Link, this would be quite handy as there is the additional button that can be utilised. Steam competes with this add-on for the middle button.

**Comment by Goretantath** [comment]
Yeah would be nice to have a toggle somewhere to swap the A and B locations of the UI. I have a Guilikit kingkong pro controller and it feels weird seeing the buttons in the wrong location on the screen.
